packages/type-safe-api/docs/developer_guides/type-safe-api/index.md

```diff
@@ -1,14 +1,18 @@
 # Type Safe API
 
-Define your APIs using [Smithy](https://smithy.io/2.0/) or [OpenAPI v3](https://swagger.io/specification/), and leverage the power of generated client and server types, infrastructure, documentation, and automatic input validation!
+The _type-safe-api_ package provides a projen project type which allows you to define an API using either [Smithy](https://smithy.io/2.0/) or [OpenAPI v3](https://swagger.io/specification/), and a construct which manages deploying this API in an API Gateway.
 
-This package vends a projen project type which allows you to define an API using either [Smithy](https://smithy.io/2.0/) or [OpenAPI v3](https://swagger.io/specification/), and a construct which manages deploying this API in API Gateway, given an integration (eg a lambda) for every operation.
+You can define your APIs using [Smithy](https://smithy.io/2.0/) or [OpenAPI v3](https://swagger.io/specification/), and leverage the power of generated client and server types, infrastructure, documentation, and automatic input validation.
 
-The project will generate "runtime" projects from your API definition in your desired languages, which can be utilised both client side for interacting with your API, or server side for implementing your API. The project also generates a type-safe CDK construct which ensures an integration is provided for every API operation.
+## How does it work?
 
-Code is generated at build time, so when you change your API model, just rebuild and you'll see your changes reflected in the generated code.
+The project will generate runtime projects from your API definition in your desired languages, which you can use both on the client side for interacting with your API, or server side for implementing your API. The project also generates a type-safe CDK construct which ensures an integration is provided for every API operation.
 
-The `TypeSafeApiProject` projen project will create the following directory structure within its `outdir`:
+!!! note
```

Comment:
Ooh do you know if this (and the other `!!!` directives) will render differently in the UI? Might be worth building locally and taking a look at the generated html documentation if you haven't already :)